.github/workflows/continuous_integration.yml

```diff
@@ -270,6 +270,12 @@ jobs:
           filters: |
             other_than_docs:
               - '!(docs/**)**'
+      - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
```

Comment:
Is the indention correct on these lines?

Reply:
Nice catch! Fixed